vdsql/__main__.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+def main():
+    import ibis
+    from visidata import main, vd
+
+    import vdsql as _  # noqa: F401
+
+    for ext in "db ddb duckdb sqlite sqlite3".split():
+        setattr(vd, f"open_{ext}", vd.open_vdsql)
+
+    for entry_point in ibis.util.backend_entry_points():
+        setattr(vd, f"openurl_{entry_point.name}", vd.open_vdsql)
```

Comment:
This means we won't be able to override these for special cases like snowflake.  Let's check with hasattr first?